setup/build_publish_docker.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
+  - name: jobName
+    type: string
+  - name: useDemands
+    type: boolean
+  - name: matrix
+    type: object
+  - name: ubuntu_tag
+    type: string
+  - name: mariner_tag
+    type: string
+
+jobs:
+  - job: ${{ parameters.jobName }}
+    timeoutInMinutes: 120
+    strategy:
+      matrix: ${{ parameters.matrix }}
+    pool:
+      ${{ if eq(parameters.useDemands, true) }}:
+        name: "blobfuse-ubn-arm64-pool"
+        demands:
+          - ImageOverride -equals ${{ parameters.matrix['Ubuntu_arm64'].agentName }}
+      ${{ if eq(parameters.useDemands, false) }}:
+        vmImage: ${{ parameters.matrix['Ubuntu_amd64'].vmImage }}
+    
+    variables:
+          - group: AZCOPY_SECRET_VAULT
+          - name: root_dir
+            value: '$(System.DefaultWorkingDirectory)'
+          - name: work_dir
+            value: '$(System.DefaultWorkingDirectory)/azure-storage-azcopy'
+
+    steps:
+      - checkout: self
+
+      - ${{ if eq(parameters.jobName, 'Set_2_Ubuntu_Mariner_ARM64') }}:
+        - task: ShellScript@2
+          inputs:
+            scriptPath: "$(work_dir)/go_installer.sh"
+            args: "$(root_dir)/ $(AZCOPY_GOLANG_VERSION)"
+          displayName: "Installing Go tools"
+
+      - ${{ if eq(parameters.jobName, 'Set_2_Ubuntu_Mariner_ARM64') }}:
+        - script: |
+            sudo apt update
```

Comment:
We can probably do this for all the cases, this should be safe to do even in the distros that we didn't need this for